service/README.md

```diff
@@ -149,8 +149,37 @@ extensions:
    ./otelcorecol validate --config=file:examples/local/otel-config.yaml
 ```
 
-## How to examine the final configuration after merging and resolving from various sources?
+## How to examine the final configuration after resolving, parsing and validating?
+
+Use `print-config` in the default mode (`--mode=redacted`) and `--feature-gates=otelcol.printInitialConfig`:
+
+```bash
+   ./otelcorecol print-config --config=file:examples/local/otel-config.yaml
+```
+
+Note that by default the configuration will only print when it is
+valid, and that sensitive information will be redacted.  To print a
+potentially invalid configuration, use `--validate=false`.
+
+## How to examine the final configuration including sensitive fields?
+
+Use `print-config` with `--mode=unredacted` and `--feature-gates=otelcol.printInitialConfig`:
 
 ```bash
-   ./otelcorecol print-initial-config --config=file:file.yaml --config=http:http://remote:8080/config --config=file:file2.yaml
-```
\ No newline at end of file
+   ./otelcorecol print-config --mode=unredacted --config=file:examples/local/otel-config.yaml
+```
+
+Note that by default the configuration will only print when it is
+valid.  To print a potentially invalid configuration, use
+`--validate=false`.
+
```

Comment:
```suggestion
```
Could we either remove this instance of this paragraph, or look at moving it elsewhere? I understand putting it in both sections, but having it repeated not long after the previous one makes it feel like an editing mishap.

We could also try to put it in a "caution" box if you think it's important enough to repeat.

Reply:
I don't think it's that important.